content/blog/nats-cli-intro.md

```diff
@@ -0,0 +1,52 @@
++++
+date = "2021-04-27"
+draft = false
+title = "Intro NATS CLI"
+author = "R.I. Piennar"
+categories = ["General", "Engineering"]
+tags = ["NATS", "CLI", "ADMIN", "JetStream"]
++++
+
+# Introducing NATS CLI
+
+Historically NATS did not have a clear obvious place to go for a CLI tool, users had various example applications
+that had to build and run manually. These tools were primarily examples of using the APIs rather than full UX CLI experience.
+
+As part of the NATS 2.2.0 release that introduced JetStream and its administration needs, we took the opportunity
+to build a full CLI tool for users and administrators alike.
+
+Below is a quick video tour of the CLI as an introduction.
+
+[![CLI Introduction](https://img.youtube.com/vi/rn5l9rQ-krM/0.jpg)](https://youtu.be/OFUjbv1ItJc)
```

Comment:
should probably upload the video to your youtube and use that link instead, and perhaps see if the website supports a inline embedding rather than pop out as here (this is all github gist supports)

Reply:
Yeah I thought that I did - so much for cut & paste

Reply:
It is the NATS youtube link - the preview image is the one you had previously.

I can make it inline viewing instead.